modules/crawler.py

```diff
@@ -254,63 +220,51 @@ def parse_meta_tags(url, soup):
         prop = meta_tag.get("property")
         content = meta_tag.get("content")
         if name and "verif" in name.lower():
-            tag_indicators.append(add_verification_tags(name, content))
+            tag_indicators.append(Indicator("1-verification_id", name + "|" + content))
         elif name and name in ["twitter:site", "fb:pages"]:
-            tag_indicators.append(add_meta_social_tags(name, content))
+            tag_indicators.append(Indicator("3-meta_social", name + "|" + content))
         elif (name or prop) and content:
             name = name or prop
             generic_metas.append(name + "|" + content)
-    tag_indicators.append(add_indicator("3-meta_generic", generic_metas))
-    return tag_indicators
+    generic_indicators = [Indicator("3-meta_generic", g_m) for g_m in generic_metas]
+    return tag_indicators + generic_indicators
 
-
-def parse_script_tags(url, soup):
+@return_empty_if_fails
+def parse_script_tags(soup) -> list[Indicator]:
     script_tags = soup.find_all("script")
-    tag_indicators = []
-    script_tags = []
+    script_tag_srcs = []
     # Iterate over the meta tags
     for script_tag in script_tags:
         # Get the name and content attributes of the meta tags
         source = script_tag.get("src")
         if source:
             match = re.search(r"/([^/]+)$", source)
             if match and match.group(1) not in script_tags:
-                script_tags.append(match.group(1))
-    tag_indicators = [add_indicator("3-script_src", script_tags)]
```

Comment:
now we return each script tag as its own indicator, instead of having them all be a list. does that make sense to you?